packages/wallets/src/features/cashier/modules/Transactions/components/TransactionsCompletedRow/components/TransactionsCompletedRowTransferAccountDetails/TransactionsCompletedRowTransferAccountDetails.tsx

```diff
@@ -24,25 +27,28 @@ const TransactionsCompletedRowTransferAccountDetails: React.FC<TProps> = ({ acco
     if (transferAccount) {
         const derivAccountType = transferAccount === wallet ? 'wallet' : 'standard';
         const accountType = transferAccount?.platform !== 'deriv' ? transferAccount.platform : derivAccountType;
+        const mt5LandingCompanyName = transferAccount === mt5Account ? mt5Account.landing_company_name : undefined;
         const displayAccountName = getAccountName({
             accountCategory: transferAccount === wallet ? 'wallet' : 'trading',
             //@ts-expect-error this needs backend typing
             accountType,
             displayCurrencyCode: transferAccount.currency_config?.display_code ?? 'USD',
-            landingCompanyName: transferAccount.landing_company_name as TWalletLandingCompanyName,
+            landingCompanyName: activeWallet?.landing_company_name as TWalletLandingCompanyName,
             mt5MarketType: transferAccount === mt5Account ? mt5Account.market_type : undefined,
         });
 
         return (
             <TransactionsCompletedRowAccountDetails
-                accountType={accountType ?? ''}
+                accountType={accountType}
                 actionType='transfer'
                 currency={transferAccount.currency ?? 'USD'}
                 displayAccountName={displayAccountName ?? ''}
                 displayActionType={`Transfer ${direction}`}
                 isDemo={Boolean(transferAccount.is_virtual)}
                 isInterWallet={transferAccount === wallet}
+                landingCompanyName={activeWallet?.landing_company_name as TWalletLandingCompanyName}
```

Comment:
Using active wallet's landing company instead of other transfer-participating acc the same way as in https://github.com/binary-com/deriv-app/blob/master/packages/wallets/src/features/cashier/modules/Transfer/components/TransferFormAccountCard/TransferFormAccountCard.tsx#L26-L30